src/Symfony/Component/Cache/Adapter/TraceableTagAwareAdapter.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Adapter;
+
+use Symfony\Component\Cache\Traits\TraceableTrait;
+
+/**
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+class TraceableTagAwareAdapter implements TagAwareAdapterInterface
```

Comment:
I agree with @xabbuh. I don't really understand the trait purpose appart from allowing us to make it `@internal` while `TraceableAdapater` is already released as "public". 
Using inheritance and adding the protected `TraceableAdapter::getPool()` method looks a better option to me for this.